### PR TITLE
Upgrade gcs to 2.2.8

### PR DESCRIPTION
**Pull request by lukasz-walkiewicz**
## Description

Replaces:  #3507 

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

### CODE REVIEW

File: plugin/trino-hive/src/main/java/io/trino/plugin/hive/gcs/GcsConfigurationProvider.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
     @Override
     public void updateConfiguration(Configuration configuration, HdfsContext context, URI uri)
     {
-        if (!uri.getScheme().equals(GoogleCloudStorageFileSystem.SCHEME)) {
```
Comment (Praveen2112) [inline plugin/trino-hive/src/main/java/io/trino/plugin/hive/gcs/GcsConfigurationProvider.java:34]: nit: Can we do a static import as a prep. commit and then maybe update the library ?

File: plugin/trino-hive/src/main/java/io/trino/plugin/hive/gcs/GoogleGcsConfigurationInitializer.java [hunk 2]
```diff
@@ -44,13 +45,13 @@ public void initializeConfiguration(Configuration config)
 
         if (useGcsAccessToken) {
             // use oauth token to authenticate with Google Cloud Storage
-            config.set(AUTH_SERVICE_ACCOUNT_ENABLE.getKey(), "false");
```
Comment (Praveen2112) [inline plugin/trino-hive/src/main/java/io/trino/plugin/hive/gcs/GoogleGcsConfigurationInitializer.java:47]: nit: Can we do this method change as a prep. commit and then maybe update the library ?

File: pom.xml [hunk 3]
```diff
@@ -1114,42 +1114,15 @@
...
-
-            <dependency>
-                <groupId>com.google.cloud.bigdataoss</groupId>
-                <artifactId>gcsio</artifactId>
-                <version>${dep.gcs.version}</version>
-                <exclusions>
-                    <exclusion>
-                        <groupId>com.google.code.gson</groupId>
-                        <artifactId>gson</artifactId>
+                        <groupId>*</groupId>
```
Comment (Praveen2112) [inline pom.xml:1120]: Should we exclude all or can we have any specific list of libraries to be excluded ?

**Comment by hashhar** [comment]
cc: @ebyhr @findepi Can you help ensure that GCS tests are run for this PR?

**Comment by lukasz-walkiewicz** [comment]
Closing as #3507 got merged.

**Pull request closed without merging**